app/(sign)/sign/[id]/components/SignPage.tsx

```
@@ -956,15 +962,24 @@ export default function SignPageComponent({
+            onClose={() => setIsModalOpen(false)}
+            onComplete={handleSignatureComplete}
+            existingText={undefined}
+          />
+        ) : (
+          <SignatureModal
+            isOpen={isModalOpen}
+            onClose={() => setIsModalOpen(false)}
+            onComplete={handleSignatureComplete}
+            existingSignature={
+              localSignatures.find((s) => s.area_index === selectedArea)
+                ?.signature_data!
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)